src/core/client/admin/components/UserRole/UserRoleChange.tsx

```diff
@@ -25,8 +28,8 @@ import UserRoleText from "./UserRoleText";
 import styles from "./UserRoleChange.css";
 
 interface Props {
-  username: string | null;
-  onChangeRole: (role: GQLUSER_ROLE_RL) => Promise<void>;
+  user: UserRoleChangeContainer_user;
```

Comment:
Do we need to send through `user` separately here? It looks like we are still only using the info about the user that is already being passed through (`role` and `moderationScopes`) to validate the role changes, unless I'm missing something? 

If we do send `user` through like this, then I think we should update to not also send through `role`, `moderationScopes`, `membershipScopes`, `moderationScoped`, and `membershipScoped` separately.